resources/english.yml

```diff
@@ -1102,4 +1102,6 @@ msg_reloaded_config: '&2Configuration successfully reloaded.'
 msg_reloaded_perms: '&2Permissions successfully reloaded.'
 msg_reloaded_db: '&2Database successfully reloaded'
 msg_reloaded_lang: '&2Language file successfully reloaded'
-msg_reset_config: '&2Config has been reset'
\ No newline at end of file
+msg_reset_config: '&2Config has been reset'
+msg_err_invalid_nation_map_color: '&cUnknown map color. Allowed colors are %s.'
+msg_nation_map_color_changed: '&bThe nation map color was changed to %s.'
```

Comment:
You've updated the English the language file, however, I don't see any updates for the other language files. Also, the version number on each language file needs to be bumped anytime there is an addition.

Reply:
> You've updated the English the language file, however, I don't see any updates for the other language files. Also, the version number on each language file needs to be bumped anytime there is an addition.

Not updating all the lang files is fine, I can do any of that later on. Having them all in place this early on can end up making lang file conflicts more difficult.

Reply:
* Bumped english lang file version
* Added comment attributing the 2 new lines to the new version